src/encoded/tests/features/forms.feature

```diff
@@ -18,6 +19,7 @@ Feature: Edit forms
     Scenario: Leaving a dirty form without saving asks for confirmation
         When I visit "/antibodies/ENCAB728YTO/#!edit"
         And I wait for 20 seconds
```

Comment:
Do we need both the 20second wait and the form wait?  Looks clunky.

Reply:
I didn't try it without the 20s wait, so I'm not sure.

Reply:
I added the 20-second wait out of sheer frustration with Travis CI.